contributing/single-sourcing-content.md

```diff
@@ -219,20 +219,81 @@ To use the component at the beginning of a sentence, add a non-breaking space ch
 <Var name="dbt" /> is awesome!
 ```
 
-## Reusing snippets of content
+## Reusing content
 
-The Snippet component allows for content to be reusable throughout the Docs. This is very similar to the existing FAQ component.
+To reuse content on different pages, you can use some techniques like partial files or snippets. Partial files, a built-in Docusaurus feature, is the recommended method over snippets.
+
+### Partial file
+
+A partial file allows you to reuse content throughout the docs. Here are the steps you can take to create and use a partial file:
+
+1. Create a new markdown partial file in the `website/snippets` directory. The file name must begin with an underscore, like `_filename.md`
+2. Go back to the docs file that will pull content from the partial file.
+3. Add the following import file: `import ComponentName from '/snippets/_this-is-your-partial-file-name.md';`
+   * You must always add an import file in that format. Note you can name `ComponentName` (a partial component) can be whatever makes sense for your purpose.
+   * `.md` needs to be added to the end of the filename. 
+4. To use the partial component, go to the next line and add `<ComponentName />`. This fetches the reusable content in the partial file
+   * Note `anyname` can be whatever makes sense for your purpose.
+
+You can also use this for more advanced use cases like reusable frontmatter.
+
+#### Partial example
+
+To create a new partial to use throughout the site, first, we will create a new markdown partial file within the snippets directory:
+
+```markdown
+/snippets/_partial-name.md
+## Header 2
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam fermentum porttitor dui, id scelerisque enim scelerisque at.
+```
+
+Now, we can add this code to a Docs file:
+
+```markdown
+Docs content here.
+
+`import SetUpPages from '/snippets/_partial-name.md';`
+
+<SetUpPages />
+
+Docs content here.
+```
+
+<details>
```

Comment:
hey @JKarlavige , ok i've reworded this based on your feedback and also added a toggle with info on how to use props to pass diff content. whenever you have a chance in the upcoming week, can you give this a 👀 to see if it's correct? thank you!